lib/protobuf/field/int64_field.rb

```diff
@@ -16,6 +16,19 @@ def self.min
         INT64_MIN
       end
 
+      ##
+      # Instance Methods
+      #
+      def acceptable?(val)
+        if val.is_a?(Integer) || val.is_a?(Numeric)
+          val >= INT64_MIN && val <= INT64_MAX
+        else
+          Integer(val, 10) >= INT64_MIN && Integer(val, 10) <= INT64_MAX
+        end
+      rescue
```

Comment:
I think we should explicitly catch the `ArgumentError` (vs rescue all) to avoid accidental typos slipping in.

Reply:
well, my intent was to not change anything except the serialization methods .... we can make this change but it would change how today's code works

Reply:
OK. We can punt this change for now.